src/core/folder.h

```diff
@@ -179,6 +183,8 @@ private Q_SLOTS:
     bool defer_content_test : 1;
 
     static std::unordered_map<FilePath, std::weak_ptr<Folder>, FilePathHash> cache_;
+    static FilePath cutFilesDirPath_;
+    static std::shared_ptr<const HashSet> cutFilesHashSet_;
```

Comment:
When cut files are set in some external application, it can be a random folder - not what is current folder in _pcmanfm-qt_.

Hence cut files are static (and "shared" for all folder instances). When this cut files' folder can be random, also `cutFilesDirPath_` is needed.

Originally it went static because I thought of possibility when cut files could be selected across more than one folder, but this was misunderstood so this design reasoning  went _old_ in some way.

Better solution might exists but I couldn't find a way how to setup locally cut files for a folder which was not loaded (and cached) yet.